php/commands/db.php

```diff
@@ -52,6 +52,10 @@ public function drop( $_, $assoc_args ) {
 	 *
 	 * [--yes]
 	 * : Answer yes to the confirmation message.
+	 * 
+	 * ## EXAMPLES
+	 *
+	 *     wp db reset --yes
```

Comment:
> Do you want me to keep the `$`? Seems like I couldn't find any examples containing it.

Yes. The point of #2769 is to update the examples so they appear closer to how you'll actually see the commands run in the real world.